backend/lib/richard_burton/flat_publication.ex

```diff
@@ -22,8 +22,9 @@ defmodule RichardBurton.FlatPublication do
     :original_title,
     :original_authors
   ]
+  @serializable_attributes [:id] ++ @external_attributes
```

Comment:
Why don't we add `:id` directly to `@external_attributes`. I believe both `serializable` and `external` attributes convey the same meaning here

Reply:
The problem here is that `@external_attributes` is used in the `changeset` function with `validate_required(@external_attributes)`, so the id would be required in that case